pkgs/servers/web-apps/netbox/default.nix

```
@@ -0,0 +1,104 @@
+{ lib, pkgs, fetchFromGitHub, python3, plugins ? [] }:
```

```suggestion
{ lib
, pkgs
, fetchFromGitHub
, python3

, plugins ? ps: [] }:
```

Make plugins consume a package set.